portfolio/src/main/webapp/tsh.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>Quote</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
Move the script tag to the end of the body tag if possble.

Loading it in head slows down the display because of script interpretation.